NEWS.md

```diff
@@ -4,6 +4,9 @@
 
 * `fixed_regex_linter()` no longer fails with regular expression pattern `"\\;"` (#1545, @IndrajeetPatil).
 
+* `lint()` no longer produces error on encountering raw HTML included in `marginfigure` code blocks 
+  from the `{tufte}` package (#796, @IndrajeetPatil).
```

Comment:
can we add some note about what's going on more generally? it may help for users experiencing a similar issue but with other engines

Reply:
I have completely rewritten the NEWS item. See if the new version is more apt.